sdk/servicebus/azure-servicebus/azure/servicebus/_common/message.py

```diff
@@ -458,6 +464,69 @@ def _is_live(self, action):
         except TypeError:
             pass
 
+    def _settle_message(
+            self,
+            settle_operation,
+            dead_letter_details=None
+    ):
+        try:
+            if not self._is_deferred_message:
+                try:
+                    self._get_settle_via_receiver_link(settle_operation, dead_letter_details)()
+                    return
+                except Exception as exception:  # pylint: disable=broad-except
```

Comment:
Is there any way we can constrain this?  or is the set of mgmt. exceptions VERY broad.

Reply:
I checked the uamqp code.
if settling through receiver failed, it would only raise RuntimeError: https://github.com/Azure/azure-uamqp-python/blob/803810434d5696b42641abfea9074069be446c29/src/message_receiver.pyx#L77